### ISSUE TITLE
Script output about applied patches is not clear

### ISSUE THREAD

**Issue by paskal** [zabbix/zabbix-patches#6]
Script output now doesn't say which patch are applied and which are failed or succeeded, it could be seen in the example below where I try to apply all available patches to Zabbix 3.0 trunk.

I guess we should add a log entry about which patch is applied at the beginning, about patch applying at the end, and some summary at the end.
Also currently there are some useless output which could be hidden.

```
~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
~/zabbix-3.0.5rc1/frontends/php ~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
patching file include/forms.inc.php
patching file items.php
Hunk #6 succeeded at 1215 (offset 5 lines).
Hunk #7 succeeded at 1290 (offset 5 lines).
~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
~/zabbix-3.0.5rc1/frontends/php ~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
patching file include/defines.inc.php
Hunk #1 succeeded at 597 (offset 7 lines).
patching file latest.php
Hunk #2 succeeded at 75 (offset -35 lines).
Hunk #3 succeeded at 86 (offset -35 lines).
Hunk #4 succeeded at 98 (offset -35 lines).
Hunk #5 succeeded at 361 (offset -35 lines).
Hunk #6 succeeded at 472 (offset -35 lines).
Hunk #7 succeeded at 605 (offset -35 lines).
Hunk #8 succeeded at 679 (offset -35 lines).
Hunk #9 FAILED at 821.
1 out of 9 hunks FAILED -- saving rejects to file latest.php.rej
~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
~/zabbix-3.0.5rc1/frontends/php ~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
patching file include/classes/macros/CMacrosResolver.php
Hunk #1 succeeded at 1158 (offset -13 lines).
patching file include/classes/macros/CMacrosResolverGeneral.php
~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
~/zabbix-3.0.5rc1 ~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
patching file src/zabbix_server/httppoller/httptest.c
~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
~/zabbix-3.0.5rc1 ~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
patching file src/zabbix_sender/zabbix_sender.c
~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
~/zabbix-3.0.5rc1/frontends/php ~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
patching file include/classes/core/CConfigFile.php
patching file include/db.inc.php
~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
~/zabbix-3.0.5rc1/frontends/php ~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
patching file app/controllers/CControllerWidgetSystemView.php
patching file dashconf.php
patching file include/blocks.inc.php
patching file include/views/monitoring.dashconf.php
~/zabbix-patches/zabbix-3.0 ~/zabbix-patches
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
~/zabbix-patches
```

**Comment by paskal** [comment]
Fixed in a0961c351d35533db2306bbf96044c3ccb208fab. New output is much clearer:
```
Applying ZBXNEXT-1456-1
patching file include/forms.inc.php
patching file items.php
Applying ZBXNEXT-1810-1
patching file include/defines.inc.php
Hunk #1 succeeded at 597 (offset 7 lines).
patching file latest.php
Hunk #2 succeeded at 75 (offset -35 lines).
Hunk #3 succeeded at 86 (offset -35 lines).
Hunk #4 succeeded at 98 (offset -35 lines).
Hunk #5 succeeded at 361 (offset -35 lines).
Hunk #6 succeeded at 472 (offset -35 lines).
Hunk #7 succeeded at 605 (offset -35 lines).
Hunk #8 succeeded at 679 (offset -35 lines).
Hunk #9 FAILED at 821.
1 out of 9 hunks FAILED -- saving rejects to file latest.php.rej
Applying ZBXNEXT-2448-1
patching file include/classes/macros/CMacrosResolver.php
Hunk #1 succeeded at 1158 (offset -13 lines).
patching file include/classes/macros/CMacrosResolverGeneral.php
Applying ZBXNEXT-2315-1
patching file src/zabbix_server/httppoller/httptest.c
Applying ZBXNEXT-1109
patching file src/zabbix_sender/zabbix_sender.c
Applying ZBXNEXT-1603
patching file include/classes/core/CConfigFile.php
patching file include/db.inc.php
Applying ZBXNEXT-2819
patching file app/controllers/CControllerWidgetSystemView.php
patching file dashconf.php
patching file include/blocks.inc.php
patching file include/views/monitoring.dashconf.php
```
During fixing extra files delivery output I've noticed mechanism was broken even before d340291ab00bcab0aefb8032b56baf53ab7e6d9c: extra file was delivered as `zabbix-2.4-ZBXNEXT-1061-1-pinset.png`, then it started to deliver as `ZBXNEXT-1061-1-pinset.png`.
Also loop is working not two times for two files but four times, but that might be not a problem. @richlv, could you please check it? I've spent a hour and couldn't find a way to fix it within current logic.

**Comment by paskal** [comment]
Original issue is resolved, I'll close this one and will create new one about incorrect files delivery.
